deploy/docs/Installation_with_Helm.md

```diff
@@ -83,11 +83,12 @@ helm upgrade --install my-release sumologic/sumologic --namespace=my-namespace -
 
 ### Installing the helm chart in Openshift platform
 The daemonset/statefulset fails to create the pods in Openshift environment due to the request of elevated privileges, like HostPath mounts, privileged: true, etc.
+Also, since the `prometheus-node-exporter` is already present in Openshift platform by default, we need to override the port being used by `prometheus-node-exporter` to `9200`.
```

Comment:
Please add the full property name and also we need to add this to all the OpenShift sections in the respective docs.

Reply:
full property name is in the installation command below and the same property is already present in the other Openshift sections in the respective docs.

Reply:
Ahh helps to read more carefully :) but I think we need to add this to other openshift sections.  eg this note so they are all consistent.

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/Non_Helm_Installation.md#installation-in-openshift-platform

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/db24c7f0ffcdaa4afb3541edbf83e121ad22d12f/deploy/docs/SideBySidePrometheus.md

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/0e63125f48a64e591b786bc0e521a966681d6a4b/deploy/docs/standAlonePrometheus.md

Just the text more than anything is what I meant.